### PR TITLE
vault_passphrase: Do not prompt for unknown validity

### PR DESCRIPTION
**Pull request by KellerFuchs**
All keys are known valid, since we pin them by fingerprint (and if admins accept unsigned updates of the git repo, we have bigger issues).

### CODE REVIEW

**Review by singlerider — COMMENTED** [review]
https://www.gnupg.org/documentation/manuals/gnupg/GPG-Configuration-Options.html

`--trust-model`
  `always`

"""Skip key validation and assume that used keys are always fully valid. You generally won’t use this unless you are using some external validation scheme. This option also suppresses the "[uncertain]" tag printed with signature checks when there is no evidence that the user ID is bound to the key. Note that this trust model still does not allow the use of expired, revoked, or disabled keys."""

I'll review further and approve.

**Comment by KellerFuchs** [comment]
> Which validation scheme is in use?

What do you mean?  How do we know that the key (fingerprints) in the `$RECIPIENTS` array are valid?

**Comment by singlerider** [comment]
I edited that comment. I didn't see your first comment before I posted. I understand now.

**Review by singlerider — APPROVED** [review]

File: vault_passphrase.sh [hunk 1]
```diff
@@ -27,7 +27,9 @@ elif [ $# -eq 1 ] && [ "$1" = "rekey" ]; then
     NEW_KEY=$(od -vAn -N32 -tx -w /dev/random)
 
     # Save the new key
-    gpg -e $(printf "%s" "${RECIPIENTS[@]/#/ -r }") -o "${NEW_KEY_FILE}" <<< "${NEW_KEY}"
+    gpg --trust-model always                        \
```
Comment (KellerFuchs) [inline vault_passphrase.sh]: Should comment why the keys are considered valid.

**Comment by KellerFuchs** [comment]
Github is silly, won't let me do a review with status “request changes”...

**Review by DeviaVir — DISMISSED** [review]
Doing this for @KellerFuchs: "Should comment why the keys are considered valid."

**Pull request merged**